docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -30,11 +30,52 @@ Modeling Strategy for the Vitamin A Deficiency Cause
 
 	Describe cause in detail
 
+Cause Hierarchy
+^^^^^^^^^^^^^^^
+.. image:: vitA_cause_hierarchy.svg
```

Comment:
The diagram looks good, except we decided that the sequela boxes should look different from the cause boxes: The sequelae should have sharp corners instead of round corners.  Sorry we still haven't documented that anywhere...

Reply:
No worries, will change sequelae boxes to sharp corner style